prometheus/exporter/gpu_exporter.go

```diff
@@ -154,6 +154,7 @@ func writeMetricsToFile(index int, logDir string) {
 func main() {
     logDir := flag.String("gpu_log_dir", "/datastorage/prom-exporters", "log written dir")   
     mode := flag.String("flag", "gpu", "gpu server")
```

Comment:
>mode [](start = 4, length = 4)

do we need a default mode? and default sleeptime?

Reply:
Yes and the entry point will override this.

Reply:
remove ;; and test docker build and run